src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BindingBuildTest.cs

```diff
@@ -227,7 +227,10 @@ public void BindingCustomJavaApplicationClass ()
 				IsRelease = true,
 			};
 			binding.AndroidClassParser = "class-parse";
-			binding.Jars.Add (new AndroidItem.EmbeddedJar (() => "$(MonoDroidInstallDirectory)\\lib\\mandroid\\android-support-multidex.jar"));
+			var multidexJar = Environment.OSVersion.Platform == PlatformID.Win32NT
+				? Path.Combine (Environment.GetFolderPath (Environment.SpecialFolder.ProgramFilesX86), "MSBuild", "Xamarin", "Android", "android-support-multidex.jar") 
```

Comment:
Outside of VS2017 (VS2015 etc) it is in Program Files (x86)..
Not sure how we run the tests under VS2017 because the locations depend on the version you run. Which sucks in my opinion.